Solutions/SemperisDirectoryServicesProtector/Data Connectors/SemperisDSP-connector.json

```diff
@@ -0,0 +1,152 @@
+        {
+            "description" : "Get all Indicators of Exposure (IoEs) tested and passed for the last 24 hours",
+            "query": "dsp_parser\n | where EventID == 9211\n | where TimeGenerated >= ago(24h)\n | summarize count() by TimeGenerated, tostring(ForestName), tostring(Domains), tostring(SecurityIndicatorName), tostring(SecurityIndicatorDescription), tostring(ResultMessage), tostring(SecurityFrameworkTags), tostring(LikelihoodOfCompromise), tostring(Score)"
+        },
+        {
+            "description" : "Get all Indicators of Exposure (IoEs) tested and passed for the last 7 days",
+            "query": "dsp_parser\n | where EventID == 9211\n | where TimeGenerated >= ago(7d)\n | summarize count() by TimeGenerated, tostring(ForestName), tostring(Domains), tostring(SecurityIndicatorName), tostring(SecurityIndicatorDescription), tostring(ResultMessage), tostring(SecurityFrameworkTags), tostring(LikelihoodOfCompromise), tostring(Score)"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Event (Semperis-DSP-Security)",
+            "lastDataReceivedQuery": "dsp_parser\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "dsp_parser\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-SemperisDSP-parser) to create the Kusto Functions alias, **dsp_parser**",
```

Comment:
Please update description as follows.
`"**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**dsp_parser**](https://aka.ms/sentinel-SemperisDSP-parser) which is deployed with the Azure Sentinel Solution."`

Reply:
Updated